src/test/resources/com/sap/oss/phosphor/fosstars/model/rating/oss/OssArtifactSecurityRatingTestVectors.yml

```diff
@@ -512,19 +517,18 @@ elements:
                 version: "V3"
                 value: 9.0
               references: []
-              resolution: "PATCHED"
+              resolution: "UNPATCHED"
               introduced: "2019-01-01"
-              fixed: "2019-01-03"
               published: "2019-01-03"
               vulnerableVersions:
                 - versionStart: 1.0.0
                   versionEnd: 1.1.0
     expectedScore:
       type: "DoubleInterval"
-      from: 4.0
+      from: 0.0
```

Comment:
The expected score here had to be re-iterated because of `UNPATCHED` or something else? Let me know.

Reply:
The test vector failed because the rating procedure produces a score ~4.54 and therefore `MODERATE` label. But the test vector expects `BAD`. I suspect that we disabled this test before we updated the thresholds for labels when we introduced the rating. I made vulnerability unpatched because it reduces the security score. That was enough to pass the test vector.

@mibo Could you please have a look at it as you are the author of these test vectors?

Reply:
@artem-smotrakov Yes, we set the test to ignore before the final thresholds were defined.
At this time the BAD artifact rating had a higher impact to the combined rating.
The changes you did are fine as for this test a BAD project security rating combined with a GOOD artifact rating should result in a BAD combined rating  ("# Test vectors for: BAD project security, GOOD artifact security").